CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contributing
+
+🏥 We're glad you want to help!
+
+## Goals of the project
+
+*  *Stay lightweight*: depend on as much of the standard library as possible; minimize external dependencies.
```

Comment:
can we fix this formatting.

Reply:
Curious as to what is wrong with this formatting?

Reply:
Talked in person - There are two whitespace characters on this line, unlike the other bullet points.